src/main/java/org/grouplens/grapht/reflect/internal/ClassInstantiator.java

```diff
@@ -153,11 +183,16 @@ public Object instantiate() throws ConstructionException {
                         }
                         throw new ConstructionException(sd, message, e);
                     }
+
                 }
             } else if (d.getInjectionPoint() instanceof NoArgumentInjectionPoint) {
                 // just invoke the method
                 Method method = ((NoArgumentInjectionPoint) d.getInjectionPoint()).getMember();
+                LogContext mdcContextInjectionPtNoArgs = LogContext.create();
                 try {
+                    method = ((NoArgumentInjectionPoint) d.getInjectionPoint()).getMember();
+                    logger.trace("Injection point method with no argument in progress {}", method);
+                    mdcContextInjectionPtNoArgs.put("org.grouplens.grapht.injectionPoint", method.toString());
```

Comment:
We should use the `toString` of the injection point object, not the method object.